src/atoms/buttons/button.tsx

```diff
@@ -225,11 +225,14 @@ const ButtonSpecialSmallNormal = Styled(Button)<Pick<ButtonProps, 'width'>>`
     height: 16px;
     fill: ${({ theme }) => theme.textShades.SHADE_MINUS_3}!important;
   };
-
+  // disabled
   ${(props) => props.disabled && css`
     background: ${props.theme.containerAndCardShades.SHADE_PLUS_1};
-    color: ${props.theme.textShades.SHADE_MINUS_1};
+    color: ${props.theme.textShades.SHADE_MINUS_1}!important;
```

Comment:
tbh i think we should double check why we need to force !important here. Can you investigate if we can get rid of this?

Reply:
yeah there is a lot of important going on in the button file in general, I think on the scg it was before we removed `fill` property from icons, so maybe a lot of the important can be gone. Will spend some time and see what I can do

Reply:
@ArinCantCode I managed to remove a couple and didn't add any additional, but we still need a bunch of !important on the svgs. I guess we could potentially add something to the backlog to take a look another look at this at some point